std/machines/test_util.asm

```diff
@@ -0,0 +1,13 @@
+/// A too-small version of std::machines::range::Byte2.
```

Comment:
A version of Byte2 that actually constrains to 0..255 because of the limited number of rows.

Reply:
done